File: ncDados/dCobrancaAutomatica.vb
Namespace nsCobranca


    Public Class ColecaoCobranca
        Inherits List(Of dCobrancaAutomatica)
    End Class
    Public Class dCobrancaAutomatica


        Private _codigocliente As Integer
        Private _crediarioid As Integer
        Private _parcelaidid As Integer
        Private _valor As Decimal
        Private _nome As String
        Private _dddcel As String
        Private _celular As String
        Private _sucesso As String
        Private _pix_code As String
        Private _data_cobranca As DateTime
        Private _datavencimento As DateTime



        Public Property CodigoCliente As Integer
            Get
                Return _codigocliente
            End Get
            Set(ByVal value As Integer)
                _codigocliente = value
            End Set
        End Property

        Public Property CrediarioId As Integer
            Get
                Return _crediarioid
            End Get
            Set(ByVal value As Integer)
                _crediarioid = value
            End Set
        End Property

        Public Property ParcelaIdId As Integer
            Get
                Return _parcelaidid
            End Get
            Set(ByVal value As Integer)
                _parcelaidid = value
            End Set
        End Property

        Public Property Valor As Decimal
            Get
                Return _valor
            End Get
            Set(ByVal value As Decimal)
                _valor = value
            End Set
        End Property

        Public Property Nome As String
            Get
                Return _nome
            End Get
            Set(ByVal value As String)
                _nome = value
            End Set
        End Property

        Public Property DDDCel As String
            Get
                Return _dddcel
            End Get
            Set(ByVal value As String)
                _dddcel = value
            End Set
        End Property

        Public Property Celular As String
            Get
                Return _celular
            End Get
            Set(ByVal value As String)
                _celular = value
            End Set
        End Property

        Public Property Sucesso As String
            Get
                Return _sucesso
            End Get
            Set(ByVal value As String)
                _sucesso = value
            End Set
        End Property

        Public Property PixCode As String
            Get
                Return _pix_code
            End Get
            Set(ByVal value As String)
                _pix_code = value
            End Set
        End Property

        Public Property DataCobranca As DateTime
            Get
                Return _data_cobranca
            End Get
            Set(ByVal value As DateTime)
                _data_cobranca = value
            End Set
        End Property

        Public Property DataVencimento As DateTime
            Get
                Return _datavencimento
            End Get
            Set(ByVal value As DateTime)
                _datavencimento = value
            End Set
        End Property


    End Class

End Namespace

File: Modelos/dCobrancaAutomatica.cs
using System;
using System.Collections.Generic;

namespace Modelos
{
    public class ColecaoCobranca : List<dCobrancaAutomatica>
    {
    }

    public class dCobrancaAutomatica
    {
        private int _codigocliente;
        private int _crediarioid;
        private int _parcelaidid;
        private decimal _valor;
        private string _nome;
        private string _dddcel;
        private string _celular;
        private string _sucesso;
        private string _pix_code;
        private DateTime _data_cobranca;
        private DateTime _datavencimento;

        public int CodigoCliente
        {
            get => _codigocliente;
            set => _codigocliente = value;
        }

        public int CrediarioId
        {
            get => _crediarioid;
            set => _crediarioid = value;
        }

        public int ParcelaIdId
        {
            get => _parcelaidid;
            set => _parcelaidid = value;
        }

        public decimal Valor
        {
            get => _valor;
            set => _valor = value;
        }

        public string Nome
        {
            get => _nome;
            set => _nome = value;
        }

        public string DDDCel
        {
            get => _dddcel;
            set => _dddcel = value;
        }

        public string Celular
        {
            get => _celular;
            set => _celular = value;
        }

        public string Sucesso
        {
            get => _sucesso;
            set => _sucesso = value;
        }

        public string PixCode
        {
            get => _pix_code;
            set => _pix_code = value;
        }

        public DateTime DataCobranca
        {
            get => _data_cobranca;
            set => _data_cobranca = value;
        }

        public DateTime DataVencimento
        {
            get => _datavencimento;
            set => _datavencimento = value;
        }
    }

}
